packages/orchestration/test/supports.ts

```diff
@@ -1 +1,62 @@
+import { makeIssuerKit } from '@agoric/ertp';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
+import { prepareLocalChainTools } from '@agoric/vats/src/localchain.js';
+import { makeFakeBankManagerKit } from '@agoric/vats/tools/bank-utils.js';
+import { makeFakeBoard } from '@agoric/vats/tools/board-utils.js';
+import { makeFakeLocalchainBridge } from '@agoric/vats/tools/fake-bridge.js';
+import type { Installation } from '@agoric/zoe/src/zoeService/utils.js';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { withAmountUtils } from '@agoric/zoe/tools/test-utils.js';
+import { makeHeapZone } from '@agoric/zone';
+import { E } from '@endo/far';
+
 export { makeFakeLocalchainBridge } from '@agoric/vats/tools/fake-bridge.js';
+
+export const orcSetup = async t => {
+  t.log('bootstrap vat dependencies');
+  const rootZone = makeHeapZone();
```

Comment:
Want to switch the root zone to a durable zone while we're at it?

Reply:
Good question. I tried but many of the fakes these tests rely on do not support durability. I've documented in a comment.

Reply:
I wondered about that. Thanks for giving it a try.